### PR TITLE
Support basename for android convertor

### PR DESCRIPTION
**Pull request by moskvin**

### CODE REVIEW

File: lib/babelish/csv2android.rb [hunk 1]
```diff
@@ -6,11 +6,14 @@ def initialize(filename, langs, args = {})
...
       @file_path = args[:output_dir].to_s
+      @output_basename = args[:output_basename].to_s
     end
 
     def language_filepaths(language)
       require 'pathname'
-      filepath = Pathname.new(@file_path) + "values-#{language.code}" + "strings.xml"
+      output_name = "strings.xml"
+      output_name = "#{@output_basename}.xml" if @output_basename
+      filepath = Pathname.new(@file_path) + "values-#{language.code}" + output_name
```
Comment (netbe) [inline lib/babelish/csv2android.rb]: ignore this

File: lib/babelish/csv2android.rb [hunk 2]
```diff
@@ -6,11 +6,14 @@ def initialize(filename, langs, args = {})
       super(filename, langs, args)
 
       @file_path = args[:output_dir].to_s
+      @output_basename = args[:output_basename].to_s
     end
 
     def language_filepaths(language)
       require 'pathname'
-      filepath = Pathname.new(@file_path) + "values-#{language.code}" + "strings.xml"
+      output_name = "strings.xml"
```
Comment (netbe) [inline lib/babelish/csv2android.rb]: prefer if else

**Comment by netbe** [comment]
@moskvin thx for the contribution, could you add a unit test for it in https://github.com/netbe/Babelish/blob/master/test/babelish/test_csv2android.rb please?

**Comment by netbe** [comment]
@moskvin Also check the travis ci tests as the change appears to break default behavior

**Comment by moskvin** [comment]
done

**Pull request merged**